UnityGLTF/Assets/UnityGLTF/Scripts/UniformMaps/MetalRoughMap.cs

```diff
@@ -86,6 +86,12 @@ public override int MetallicRoughnessXTexCoord
 			set { return; }
 		}
 
+		public override double RoughnessFactor
+		{
+			get { return _material.GetFloat("_Glossiness"); }
+			set { _material.SetFloat("_Glossiness", (float)value); }
+		}
```

Comment:
Does this render correctly?  Glossiness is the inverse of roughness.  I do see that MetallicRoughnessTexture in the cs files refers to MetallicGlossMap in the shaders.  Is this an optimization to use the same variable to hold different values for the different shader variants?  It seems odd to me.

Reply:
It is very odd.  Im not sure exactly what the deal is.  In the PbrMetallicRoughness.shader file, it has
`_Glossiness("Roughness Factor", Range(0.0, 1.0)) = 1.0`
as the shader parameter.  I tested in babylonjs (see pictures) and it renders the same in the editor with the change.

Best I can tell, is that PbrMetallicRoughness.shader and PbrSpecularGlossiness.shader both use UnityStandardInput.cginc, which uses _Glossiness.  These shaders also use a couple unity internal cginc files, which I assume require the _Glossiness parameter as well.  Though without digging through it, I can only speculate.

But, like I said, it works.

babylonjs (left) vs Unity (right)
![image](https://user-images.githubusercontent.com/5903547/56510850-e3922080-64df-11e9-9be8-a10230310151.png)